src/BinSkim.Driver/AnalyzeCommand.cs

```diff
@@ -134,6 +137,24 @@ public override int Run(AnalyzeOptions analyzeOptions)
                 : result;
         }
 
+        internal static SarifLog ReadSarifLog(IFileSystem fileSystem, AnalyzeOptions analyzeOptions)
```

Comment:
>fileSystem [](http://example.com/codeflow?start=58&length=10)

the fileSystem is passed from this.FileSystem
it is used in one of the version only,
I see SarifLog.Load has a override with Stream as parameter, not sure if that can be used.

Reply:
Today, there is not a way to pass a filesystem to the sariflog.